tracking_manager/models/ir_model.py

```diff
@@ -0,0 +1,206 @@
+# Copyright (C) 2022 Akretion (<http://www.akretion.com>).
+# @author Kévin Roche <kevin.roche@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from ast import literal_eval
+
+from odoo import api, fields, models, tools
+
+
+class IrModel(models.Model):
+    _inherit = "ir.model"
+
+    active_custom_tracking = fields.Boolean()
+    tracked_field_count = fields.Integer(compute="_compute_tracked_field_count")
+    automatic_custom_tracking = fields.Boolean(
+        compute="_compute_automatic_custom_tracking",
+        readonly=False,
+        store=True,
+        help=("If tick new field will be automatically tracked if the domain match"),
```

Comment:
```suggestion
        help=("If marked, the fields matching the matched by the domain below will be automatically tracked for this model."),
```
perhaps this slightly improves the help description?